nchs_mortality/delphi_nchs_mortality/pull.py

```diff
@@ -42,25 +53,44 @@ def pull_nchs_mortality_data(token: str, map_df: pd.DataFrame, test_mode: str):
         # Pull data from Socrata API
         client = Socrata("data.cdc.gov", token)
         results = client.get("r8kw-7aab", limit=10**10)
-        df = pd.DataFrame.from_records(results).rename(
-                {"start_week": "timestamp"}, axis=1)
+        df = pd.DataFrame.from_records(results)
+        # drop "By Total" rows
+        df = df[df["group"].transform(str.lower) == "by week"]
     else:
         df = pd.read_csv("./test_data/%s"%test_mode)
 
-    # Check missing start_week == end_week
-    try:
-        assert sum(df["timestamp"] != df["end_week"]) == 0
-    except AssertionError as exc:
-        raise ValueError(
-            "end_week is not always the same as start_week, check the raw file"
-        ) from exc
+    df = standardize_columns(df)
+
+    if "end_date" in df.columns:
+        # Check missing week_ending_date == end_date
+        try:
+            assert sum(df["week_ending_date"] != df["end_date"]) == 0
```

Comment:
If I understand what is going on here, it seems it would be better to use `all()` or `any()` rather than `sum()`.